skaha/src/main/java/org/opencadc/skaha/session/PostAction.java

```diff
@@ -467,9 +471,20 @@ public void attachSoftware(String image, List<String> params) throws Exception {
         log.debug("Using parameter: " + param);
         
         String uniqueID = new RandomStringGenerator(8).getID();
```

Comment:
Could you please change this uniqueID to be length 3 rather than 8.  It will save us some space.

Reply:
Done